server/go/README.md

```diff
@@ -0,0 +1,30 @@
+# Asherah Server - Go
+
+## Running the server
+This assumes mysql is running on localhost and a preexisting asherah database
+
+```console
+$ go build -o server main.go
+$ ./server -s /tmp/appencryption.sock \
+    --service=example \
+    --product=servicelayer \
+    --expire-after=60m \
+    --check-interval=10m \
+    --metastore=rdbms \
+    --conn='root:my-secret-pw@tcp(0.0.0.0:3306)/asherah' \
+    --kms=static
+```
+
+Arguments can also be supplied using environment variables
+
```

Comment:
be good to add a comment here stating the possible values for `ASHERAH_METASTORE_MODE` and `ASHERAH_KMS_MODE`

Reply:
possible values are explained in the `Configuring the server` section. should be fine if we don't add them here IMO

Reply:
The `Configuring the server` section was added in response to @nikoo28 's suggestion :)